projects/media-viewer/src/lib/viewers/viewer-event.service.spec.ts

```diff
@@ -1,12 +1,12 @@
-import {ViewerEventsService} from './viewer-events.service';
+import {ViewerEventService} from './viewer-event.service';
 import {Subscription} from 'rxjs';
```

Comment:
spacing

Reply:
I didn't want to make unnecessary formatting changes as part of this PR.... it's already big enough